molecule/default/tests/test_create_and_assign_floating_ip.py

```diff
@@ -0,0 +1,83 @@
+import utils
+import os
+import pytest
+import testinfra.utils.ansible_runner
+
+testinfra_hosts = testinfra.utils.ansible_runner.AnsibleRunner(
+    os.environ['MOLECULE_INVENTORY_FILE']).get_hosts('os-infra_hosts')[:1]
+
+utility_container = ("lxc-attach -n $(lxc-ls -1 | grep utility | head -n 1) "
+                     "-- bash -c '. /root/openrc ; ")
+
+random_str = utils.generate_random_string(6)
+instance_name = "test_instance_{}".format(random_str)
+image_name = 'Cirros-0.3.5'
+getway_net = 'GATEWAY_NET'
+private_net = 'PRIVATE_NET'
+flavor = 'm1.tiny'
+floating_ip_name = None
+
+
+def create_instance_from_image(host):
```

Comment:
@phongdly wouldn't this require a `run_on_host` parameter? Otherwise it is using the infratest `host` fixture rather than the `host` object passed in on line 59 below. I don't know if this would cause unexpected behavior or not.

Reply:
It won't create any error on this test case since they are all running on the same host, but it is definitely better to use `run_on_host` to make it cleaner. 
Updated the PR